### PR TITLE
Fix different alerts triggers creating alerts with different subscription data

### PR DESCRIPTION
**Pull request by gagantrivedi**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/deployment/locally-api#pre-commit) to check linting
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?

## Changes
* Fix different alerts triggers generating alert emails with different subscription data

## How did you test this code?

Covered by unit test cases

### CODE REVIEW

**Comment by gagantrivedi** [comment]
Depends on https://github.com/Flagsmith/flagsmith/pull/1541

File: api/organisations/alerts.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from users.models import FFAdminUser
+
+ALERT_EMAIL_MESSAGE = (
+    "Organisation %s has used %d seats which is over their plan limit of %d (plan: %s)"
+)
+ALERT_EMAIL_SUBJECT = "Organisation over number of seats"
+
+
+def send_org_over_limit_alert(organisation):
+    subscription_metadata = get_subscription_metadata(organisation)
```
Comment (matthewelwell) [inline api/organisations/alerts.py]: I appreciate this makes it a bit more work but should we just provide the subscription metadata as an argument to this function? Seems unnecessary to get the metadata again here given that we already know it? Also, should we take this opportunity to make this a task for the task processor?

Comment (gagantrivedi) [reply api/organisations/alerts.py]: > but should we just provide the subscription metadata as an argument to this

I don't think it still makes sense (considering it's now a task and argument needs to be dump/loadable)?. I have updated this to accept organisation_id as the only argument; we can update it to pass all the needed data directly and save two calls, but I think that will make the code a bit less readable with the function accepting more than 2 arguments?

Comment (matthewelwell) [reply api/organisations/alerts.py]: Agreed.

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**